app/templates/pdf/orders.html

```diff
@@ -0,0 +1,131 @@
+        }
+
+        .total-row {
+            margin-top: 25px;
+        }
+
+        .order-total {
+            margin-right: 50px;
+        }
+
+        tr.padded td{
+        padding-top : 40px;
+        }
+    </style>
+</head>
+<body id="body">
+<h3 style="text-align:center;">{{ event.name }} - {{ ("Order List") }}</h3>
+    <br>
+    <table class="table with-datatable" id="orders-table" width="100%">
+        <thead>
+        <tr>
+            <th>
+                {{ ("Order") }}
+            </th>
+            <th>
+                {{ ("Order Details") }}
+            </th>
+            <th>
+                {{ ("User") }}
+            </th>
+            <th>
+                {{ ("User Details") }}
+            </th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for order in orders %}
+            {% if order.status != "deleted" %}
+                <tr class="padded">
+                    <td><br>
+                            Order Number    :<br>
+                            Order By        :<br>
+                            Order Status    :<br>
+                            Quantity        :<br>
+                            Total Amount    :<br>
+                            Payment Mode    :<br>
+                            Discount Code   :<br>
+                        {% if order.status == 'completed' %}
+                            Order Completed :<br>
+                        {% else %}
+                            Order Placed    :<br>
+                        {% endif %}
+                    </td>
+                    <td><br>
+                            <strong>{{ order.get_invoice_number() }}</strong><br>
+                            {% if order.user and order.user.first_name and order.user.last_name %}
+                                {{ order.user.first_name + order.user.last_name }}
+                            {% elif order.user %}
+                                {{ order.user._email }}
+                            {% else %}
+                                {{ ('Information unavailable') }}
+                            {% endif %}<br>
+                            {% if order.status == 'completed' %}
+                                {{ order.status | capitalize }}
+                            {% elif order.status == 'pending' or order.status == 'initialized' %}
+                                {{ ("Pending") }}
+                            {% elif order.status == 'placed' %}
+                                {{ order.status | capitalize }}
+                            {% elif order.status == 'cancelled' %}
+                                {{ order.status | capitalize }}
+                            {% else %}
+                                {{ order.status | capitalize }}
+                            {% endif %}<br>
+                            {{ order.get_tickets_count() }}<br>
+                            {{ event.payment_currency | currency_symbol }}{{ order.amount | money }}<br>
```

Comment:
@srv-twry the Codacy issues are invalid since the filters are used in the template